source/renderer/app/components/notifications/RestoreNotification.js

```diff
@@ -38,8 +44,9 @@ export default class RestoreNotification extends Component<Props> {
     return (
       <div className={restoreNotificationClasses}>
         <span className={styles.text}>
-          {intl.formatMessage(messages.activeRestoreMessage)}: {restoreProgress}
-          %
+          {intl.formatMessage(messages.activeRestoreFirstMessage)}{' '}
```

Comment:
@DeeJayElly please update this part so that we have a single translation message which takes in the percentage as a variable. There are many examples of this such as: https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/i18n/locales/en-US.json#L36

Reply:
Fixed